tests/test_missing.py

```diff
@@ -0,0 +1,78 @@
+import os
+from this import d
+import numpy as np
+import shutil
+import tempfile
+import unittest
+
+from activestorage.active import Active
+from activestorage.dummy_data import *
+
+
+class TestActive(unittest.TestCase):
+    """ 
+    Test basic functionality
+    """
+
+    def setUp(self):
+        """ 
+        Ensure there is test data
+        """
+        self.temp_folder = tempfile.mkdtemp()
+
+    def tearDown(self):
+        """Remove temp folder."""
+        shutil.rmtree(self.temp_folder)
+
+
+    def _doit(self, testfile):
```

Comment:
```suggestion
    def _vanilla_mean(self, testfile):
```

Reply:
At some point we are going to expand these tests to cover all the supported operations, we don't want one of these for each operation, so if you want to change the name, call it `_do_test`. I actually think `_doit` is fine though!

Reply:
OK, Capt. Picard :grin: